tests/plugins/contextmenu/manual/enabledNativeSpellChecker.html

```diff
@@ -0,0 +1,16 @@
+<h2>Spell Check Manual Test - Right Click should use native browser spell check when disableNativeSpellChecker is false (native spell checker enabled).</h2>
+<textarea id="editor1">&lt;p&gt;The more a thing tends to be permanent, the more it tends to be lifeleess.&lt;/p&gt;</textarea>
+
+<script>
+	// No console support on mobiles (without connecting to another device).
```

Comment:
From what I understand this test doesn't require console so this comment is a bit confusing. Still as context menu is not really there on mobiles, I think the test itself still should be ignored there.